benchmarks/reframe_config.py

```diff
@@ -423,27 +429,23 @@ def spack_root_to_path():
                         'reframe %(version)s|'
                         '%(check_info)s|'
                         'jobid=%(check_jobid)s|'
-                        '%(check_perf_var)s=%(check_perf_value)s|'
                         'num_tasks=%(check_num_tasks)s|'
                         'num_cpus_per_task=%(check_num_cpus_per_task)s|'
                         'num_tasks_per_node=%(check_num_tasks_per_node)s|'
+                        '%(check_perfvalues)s|'
+                        'spack_spec=%(check_spack_spec)s|'
+                        'env_vars=%(check_env_vars)s'
+                    ),
+                    'format_perfvars': (
+                        '%(check_perf_var)s=%(check_perf_value)s|'
                         'ref=%(check_perf_ref)s|'
-                        'lower=%(check_perf_lower_thres)s|'
-                        'upper=%(check_perf_upper_thres)s|'
+                        'lower=%(check_perf_lower)s|'
+                        'upper=%(check_perf_upper)s|'
                         'units=%(check_perf_unit)s|'
-                        'spack_spec=%(check_spack_spec)s|'
-                        'variables=%(check_variables)s'
```

Comment:
Specifically in these lines. Do we really need the `XXXXX=%(XXXXX)`. The new perf log header will print this information anyway

Reply:
Good point.  I'll defer the decision to @ilectra and @pineapple-cat, as they are more interested in the exact format of the perflogs, but I think you're right that we should be able to get rid of them now.

Reply:
And I agree also with changing `%(check_perf_var)s=%(check_perf_value)s` to `%(check_perf_value)s`.

Reply:
I think it might be worth having separate columns for variables and values like so `perf_var=%(check_perf_var)s|perf_val=%(check_perf_value)s` because we can have multiple figures of merit, though I'm not sure what the default format is for displaying the variables and values when there are more than one. Aside from that, feel free to push this since I'll be making more changes to the format anyway, starting with getting all fields to conform to a `name=value` pattern.